modin/pandas/test/test_io.py

```diff
@@ -1844,7 +1851,7 @@ def test_read_excel_all_sheets(self, make_excel_file):
             df_equals(modin_df.get(key), pandas_df.get(key))
 
     @pytest.mark.xfail(
-        Engine.get() != "Python",
+        Engine.get() != "Python" and StorageFormat.get() != "Hdk",
```

Comment:
Doesn't the base implementation work for `HDK`?

Reply:
It defaults to pandas here - https://github.com/modin-project/modin/blob/master/modin/core/io/io.py#L283

Reply:
Oops, it seemed to me that it does not work for `HDK`, but on the contrary it works for `HDK` :), sorry